mne/io/meas_info.py

```diff
@@ -1414,6 +1414,8 @@ def write_meas_info(fid, info, data_type=None, reset_range=True):
         write_string(fid, FIFF.FIFF_PROJ_NAME, info['proj_name'])
     if info.get('meas_date') is not None:
         write_int(fid, FIFF.FIFF_MEAS_DATE, info['meas_date'])
+    else:
+        write_int(fid, FIFF.FIFF_MEAS_DATE, DATE_NONE)
```

Comment:
@bloyl can you help me understand why the writer should be fixed? It seems normal not to write anything if meas_date is None. Is it the reader that does not set the date to None when it's not present in the file on disk?

Reply:
over my morning coffee, I agree the writer shouldn't be fixed. It seemed like the most straightforward at the time.

The current fix will have non mne-python readers showing strange dates. or faulting trying to convert DATA_NONE to a timestamp.